### PR TITLE
apply changes from yesterdays review meeting

### PR DESCRIPTION
**Pull request by callumrollo**
Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [x] Fix of some error, inconsistency, unforeseen limitation.

This addressed the lion's share of todo items from yesterday documented in #215 

@emmerbodc  @vturpin please review at your earliest convenience. Once this is merged I'll assign and resolve the remaining items in #215

### CODE REVIEW

File: OG_Format.adoc [hunk 1]
```diff
@@ -442,22 +434,25 @@ PLATFORM_DEPTH_RATING:long_name = “depth limit in meters of the glider for thi
...
 
  |highly desirable
+| *ex.:* "7460"
 |PLATFORM_MAKER a|
 string PLATFORM_MAKER
 
 PLATFORM_MAKER:long_name = “glider manufacturer”;
 
-PLATFORM_MAKER:platform_maker_vocabulary = “”;
+PLATFORM_MAKER:platform_maker_vocabulary = “https://vocab.nerc.ac.uk/collection/B75/current/”;
```
[the remark this answers is not in the document]
Comment (vturpin) [reply OG_Format.adoc:452]: see issue #220

File: OG_Format.adoc [hunk 2]
```diff
@@ -665,21 +655,23 @@ Where <sensor_type> is a controlled vocabulary: see vocabulary guidelines
...
+sensor_model: “” - mandatory
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_maker: “” - highly desirable 
+sensor_model_vocabulary: “https://vocab.nerc.ac.uk/collection/L22/current” - mandatory
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_maker_vocabulary: “” - highly desirable 
+sensor_maker: “” - highly desirable
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_serial_number: “” - highly desirable 
+sensor_maker_vocabulary: “https://vocab.nerc.ac.uk/collection/L35/current” - highly desirable
```
Comment (vturpin) [inline OG_Format.adoc:670]: see above

File: OG_Format.adoc [hunk 3]
```diff
@@ -665,21 +655,23 @@ Where <sensor_type> is a controlled vocabulary: see vocabulary guidelines
...
+sensor_type_vocabulary: “https://vocab.nerc.ac.uk/collection/L05/current/” - mandatory
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_model: “” - mandatory 
+long_name: “” - mandatory
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_model_vocabulary: “” - mandatory 
+sensor_model: “” - mandatory
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_maker: “” - highly desirable 
+sensor_model_vocabulary: “https://vocab.nerc.ac.uk/collection/L22/current” - mandatory
```
Comment (vturpin) [inline OG_Format.adoc:666]: see above

File: OG_Format.adoc [hunk 4]
```diff
@@ -665,21 +655,23 @@ Where <sensor_type> is a controlled vocabulary: see vocabulary guidelines
 
 string SENSOR_<sensor_type>_<sensor_serial_number>
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_type_vocabulary = “” - mandatory
+attributes:
 
-SENSOR_<sensor_type>_<sensor_serial_number>:long_name: “” - mandatory 
+sensor_type_vocabulary: “https://vocab.nerc.ac.uk/collection/L05/current/” - mandatory
```
Comment (vturpin) [inline OG_Format.adoc:660]: see above

**Review by vturpin — APPROVED** [review]
Thanks Callum for the hard work. 

I am happy to approve this PR as it is. It clarifies the document more and more.  So you can close the discussion I've raised to merge.

However, a discussion is needed about some *_vocabularies, as I think we are missing the m2m benefit if we only link to the uri of the collection and not the uri of the term itself.
I'll open an issue about this not to forget the point.

We will need a second round of review anyway.

File: OG_Format.adoc [hunk 5]
```diff
@@ -403,37 +390,42 @@ mandatory
...
+ |mandatory
+| *ex.:*  '6801706'
+
 |PLATFORM_MODEL a|
 string PLATFORM_MODEL
 
 PLATFORM_MODEL:long_name: “model of the glider”;
 
-PLATFORM_MODEL:platform_model_vocabulary = “”;
+PLATFORM_MODEL:platform_model_vocabulary = “https://vocab.nerc.ac.uk/collection/B76/current/”;
```
[the remark this answers is not in the document]
Comment (vturpin) [reply OG_Format.adoc:410]: I've created a issue with regard to this topic. I'll close the conversation to facilitate merging.

File: OG_Format.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (vturpin) [reply OG_Format.adoc:452]: I've created a issue with regard to this topic. I'll close the conversation to facilitate merging.

File: OG_Format.adoc (hunk 4 above)
Comment (vturpin) [reply OG_Format.adoc:660]: I've created a issue with regard to this topic. I'll close the conversation to facilitate merging.

File: OG_Format.adoc (hunk 3 above)
Comment (vturpin) [reply OG_Format.adoc:666]: I've created a issue with regard to this topic. I'll close the conversation to facilitate merging.

File: OG_Format.adoc (hunk 2 above)
Comment (vturpin) [reply OG_Format.adoc:670]: I've created a issue with regard to this topic. I'll close the conversation to facilitate merging.

**Review by emmerbodc — APPROVED** [review]

**Pull request merged**